hack/coverage.sh

```diff
@@ -4,7 +4,8 @@ GINKGO="$1"
 TARGETCOVERAGE="$2"
 
 COVERAGE_FILE=cover.out
-GINKGO_COVERAGE_ARGS="-cover -coverprofile=${COVERAGE_FILE} -outputdir=. --skipPackage ./vendor"
+
+GINKGO_COVERAGE_ARGS="-cover -coverprofile=${COVERAGE_FILE} -outputdir=.  --skipPackage ./vendor"
```

Comment:
Can we drop the unnecessary whitespace changes please?

Reply:
fixed